_config.yml

```
@@ -1,8 +1,8 @@
 # Site settings
 remote_theme: pmarsceill/just-the-docs
 title: Fyde Docs
-baseurl: /docs/
-url: https://fydeinc.github.io
+baseurl: /docs
+url: https://fydeinc.github.io/docs
```

I don't think the url is supposed to have the path (/docs) in it. Example here https://github.com/pmarsceill/just-the-docs/blob/master/_config.yml

```suggestion
url: https://fydeinc.github.io
```

---

Fixed